### PR TITLE
DM-41664: Rename prompt_prototype package and containers

### PR DESCRIPTION
**Pull request by kfindeisen**
This PR updates the Phanalx configuration for `prompt-proto-service-*` to point to the new package name and the new container names. It makes no attempt to rename the service itself (deferred to [DM-41665](https://jira.lsstcorp.org/browse/DM-41665)).

This PR should be merged after lsst-dm/prompt_processing#96, though most of the changes that affect this PR are already visible.

### CODE REVIEW

**Comment by kfindeisen** [comment]
`CI / Test deploy` is not expected to pass until lsst-dm/prompt_processing#96 merges; until then, there is no `latest` tag for the new containers but branch tags can still be deployed through the Argo GUI.

File: applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ prompt-proto-service:
     tag: latest
 
   instrument:
-    pipelines: (survey="SURVEY")=[${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml]
+    pipelines: (survey="SURVEY")=[${PROMPT_PROCESSING_DIR}/pipelines/LATISS/ApPipe.yaml]
```
Comment (hsinfang) [inline applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml:14]: The `${RUBIN_INSTRUMENT}` --> `LATISS` change seems irrelevant here.  Though I don't mind the change and we probably want it anyway.

**Review by hsinfang — APPROVED** [review]

File: applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml (hunk 1 above)
Comment (kfindeisen) [reply applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml:14]: Sorry, I can at least split it into its own commit.
